guides/common/modules/proc_changing-the-method-the-bootstrap-script-uses-to-download-the-consumer-rpm.adoc

```diff
@@ -20,7 +20,7 @@ Use `--download-method` to change the download method from HTTP to HTTPS.
 --download-method https
 ----
 
-* On {RHEL} 6, or 7, enter the following command:
+* On {EL} 6, or 7, enter the following command:
```

Comment:
```suggestion
* On {EL} 6 or 7, enter the following command:
```

Reply:
This appears to be a repeated thing, introduced in 39fc9a7bd4cdfb841e26e5d3414d0431d87f7e02. Can we consider it out of scope?
```console
$ rg '6, or 7'
guides/common/modules/proc_registering-a-host-for-content-management-only.adoc
17:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_creating-a-domain-for-a-host-during-registration.adoc
22:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_migrating-a-host-from-one-server-to-another-server.adoc
20:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_migrating-a-host-from-red-hat-network.adoc
25:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_providing-the-hosts-ip-address.adoc
22:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_providing-an-alternative-fqdn-for-the-host.adoc
34:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_changing-the-method-the-bootstrap-script-uses-to-download-the-consumer-rpm.adoc
23:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_enabling-remote-execution-on-the-host.adoc
22:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_registering-a-host-using-the-bootstrap-script.adoc
92:* On {EL} 6, or 7, enter the following command:

guides/common/modules/proc_registering-a-host-without-puppet.adoc
21:* On {EL} 6, or 7, enter the following command:
```

Reply:
OK, it should be relatively easy to fix this afterwards.